src/cdk/drag-drop/drag-ref.ts

```diff
@@ -732,7 +735,7 @@ export class DragRef<T = any> {
       // from the DOM completely, because iOS will stop firing all subsequent events in the chain.
       element.style.display = 'none';
       this._document.body.appendChild(parent.replaceChild(placeholder, element));
-      getPreviewInsertionPoint(this._document).appendChild(preview);
+      getPreviewInsertionPoint(this._document, this.previewContainer).appendChild(preview);
```

Comment:
I don't think the logic for failling back to the configured preview container belongs in `getPreviewInsertionPoint`. It would be cleaner by:

1. Renaming `getPreviewInsertionPoint` to something like `getGlobalPreviewInsertionPoint`.
2. Changing the logic here to:

```
const previewContaier = this.previewContainer || getGlobalPreviewInsertionPoint();
```

Reply:
yeah completely agree - simpler, cleaner 👍